script/utils/utils.ts

```
@@ -907,3 +923,74 @@ Selectors: ${selectors.length} functions
+  network: string
+  privateKey?: string
+}): Promise<void> {
+  const calldata = await encodeDiamondCutCalldata(
+    options.facetName,
+    options.facetAddressHex
+  )
+
+  if (isTronNetworkKey(options.network)) {
+    const { runPropose } = await import(
+      '../deploy/tron/propose-to-safe-tron'
+    )
```

ESLint

[error] Dependency cycle via ../../demoScripts/utils/demoScriptHelpers:29

(import/no-cycle)